src/Magnum/Audio/Listener.h

```diff
@@ -0,0 +1,222 @@
+## Active Listener
+
+There can only be at the most *one* active listener at a given time, i.e. the
+one on which @ref Listener::update() was called last. This is because OpenAL
+only supports notion of one listener. Having multiple @ref Listener2D or
+@ref Listener3D instances can still be useful for conveniently switching
+between them for cinematics for example.
+
+@ref AbstractObject::setClean() will not affect inactive listeners.
+
+## Sound Transformation
+
+@ref Listener::setSoundTransformation() enables you to set a transformation
+matrix which is applied to the listeners orientation and position before passed
+onto OpenAL. This can be used for rotating two dimensional scenes as planes
+into the three dimensional audio space or even scaling the audio scene to match
+a certain world scale. In the later case you might want to instead consider
+@ref Renderer::setSpeedOfSound().
+
+-   @ref Listener2D
+-   @ref Listener3D
+
+@see @ref Audio::Renderer, @ref Playable, @ref PlayableGroup
+*/
+template <UnsignedInt dimensions> class Listener: public SceneGraph::AbstractFeature<dimensions, Float> {
+        friend class ActiveListenerHolder;
+
+    public:
+
+        /**
+         * @brief Constructor
+         * @param object    Object this listener belongs to
+         *
+         * Creates a listener with a forward vector of `{0.0f, 0.0f, 1.0f}` and
+         * up vector of `{0.0f, 1.0f, 0.0f}`. These vectors cannot be changed,
+         * the listeners orientation and translation can be instead affected by
+         * `object` or via @ref Listener::setSoundTransformation().
+         * @see @ref setGain()
+         */
+        explicit Listener(SceneGraph::AbstractObject<dimensions, Float>& object):
+            SceneGraph::AbstractFeature<dimensions, Float>(object),
+            _gain{1.0f}
+        {
+            SceneGraph::AbstractFeature<dimensions, Float>::setCachedTransformations(SceneGraph::CachedTransformation::Absolute);
+        }
+
+        /**
+         * @brief Sound transformation
+         * @see @ref Listener::setSoundTransformation()
+         */
+        const Matrix4& soundTransformation() const {
+            return _soundTransformation;
+        }
+
+        /**
+         * @brief Set sound transformation
+         * @param soundTransformation Transformation for transforming from
+         *        world to listener space
+         * @return Reference to self (for method chaining)
+         * @see @ref Listener::soundTransformation()
+         */
+        Listener<dimensions>& setSoundTransformation(const Matrix4& soundTransformation) {
+            _soundTransformation = soundTransformation;
+            this->object().setDirty();
+            return *this;
+        }
+
+        /**
+         * @brief Update the listener
+         * @param groups Groups to update
+         *
+         * Calls @ref SceneGraph::AbstractObject::setClean() on its parent
+         * object and all objects of the @ref Playable s in the group.
```

Comment:
Maybe mention that this also sets the listener as active and updates renderer configuration.